src/CSModule.sol

```diff
@@ -1454,7 +1405,8 @@ contract CSModule is
         }
         stuckValidatorsCount = no.stuckValidatorsCount;
         refundedValidatorsCount = no.refundedValidatorsCount;
-        stuckPenaltyEndTimestamp = no.stuckPenaltyEndTimestamp;
+        // @dev unused in CSM
+        stuckPenaltyEndTimestamp = 0;
```

Comment:
doesn't make sense to set value to 0 here. it is already zero by default

Reply:
i'd prefer to have it explicitly